src/Apps/Order/Routes/Payment/PaymentContent.tsx

```diff
@@ -218,9 +199,9 @@ const RadioWithLabel = styled(BorderedRadio)`
 `
 
 /*
-returns all available payment methods, by checking relevant feature flags
-TODO: when ACH and wire_transfer FFs is removed, this function can be removed and radios can be moved to PaymentContent
-*/
+ * returns all available payment methods in the form of BorderRadio
```

Comment:
I'd assume yes, since payment methods are determined by the API now, we'll always need to check it per order (unless all methods are available for all orders at some point, which I don't see happening).